Create 2014-02-14-bdferrcodingbat2.md

bdferr opened the pull request:
This is part 2 of my CodingBat exercises; apparently I have fixed the cause of the previous building error, because it can now be viewed here: http://bdferr.github.io/spring2014/2014/02/14/bdferrcodingbat2a.html

Commit:
Added a test post.

Commit:
Create 2014-02-14-bdferrcodingbat2.md

This is part 2 of my CodingBat exercises.

Commit:
Create 2014-02-14-bdferrcodingbat2a.md

Maybe I put this in the wrong folder the first time?

gerbal commented:
It's failing because one of your posts is in the root directory, not the posts directory

Commit:
Delete 2014-02-14-bdferrcodingbat2.md

This somehow ended up in the wrong directory; it should be in the posts directory.

Commit:
Changed the date of my test post.

Commit:
Merge branch 'gh-pages' of https://github.com/silshack/spring2014 into gh-pages

Commit:
Merge branch 'gh-pages' of https://github.com/bdferr/spring2014 into gh-pages

Commit:
Merge branch 'gh-pages' of https://github.com/silshack/spring2014 into gh-pages

eah13 commented:
@bdferr nice in-depth walk-thru of your sum67 post.  I think the mental solution you came up with might be difficult to render the way you tried to do it.  Sometimes creativity needs to be bounded by a thorough knowledge of a language and its idioms.  We'll definitely discuss this problem next class.
